pkg/generator/gitignore_tmpls.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generator
+
+const tmpGitignoreTmpl = `_output`
```

Comment:
Can we let the `.gitignore` at the project root to ignore the `tmp/_output` folder instead?

e.g:
```
# Folders
tmp/_output/
```